app/facades/quote_facade.rb

```diff
@@ -0,0 +1,14 @@
+require "./app/services/quote_service"
+require "./app/services/quote_service"
+
+class QuoteFacade
+
+  def self.find_quotes(keyword)
+    parsed_data = QuoteService.get_quotes(keyword)
+    quotes = parsed_data[:results]
+  
+    quotes.map do |quote_data|
+      Quote.new(quote_data)
+    end
+  end
+end
```

Comment:
👍 I know from your comment that you know about total count. But, this would be a good place to pull that out. You could even return both things at once: 

```ruby
parsed_data = QuoteService.get_quotes(keyword)
quotes = parsed_data[:results]
quotes = quotes.map do |quote_data|
      Quote.new(quote_data)
    end
total_count, quotes = [parsed_data[:totalCount],quotes]